### DISCUSSION TITLE
Decimal precission of numbers

### DISCUSSION THREAD

**Discussion by ghost** [handsontable/hyperformula#1095] [API Questions]
Is there any way to set a parameters defining precision of decimal places across all the cells?
I try to copy Excel's behaviour. Initially I hoped 'precisionRounding' will do that, but:

- I struggled to make it work, and I gave up because
- Documetnation suggests it does something rather different

**Comment by sequba** [comment]
Hi @bjorngrundtalf, thank you for reaching out to us.

We want to understand your issue better. Can you give us some examples of calculations that result in different (less accurate) values in HyperFormula and Excel?
